hypertrace-graphql-span-processing-schema/src/main/java/org/hypertrace/graphql/spanprocessing/schema/rule/ExcludeSpanRuleInfo.java

```diff
@@ -16,21 +17,26 @@ public interface ExcludeSpanRuleInfo {
 
   @GraphQLField
   @GraphQLName(NAME_KEY)
+  @GraphQLDescription("Exclude span rule name")
   @GraphQLNonNull
   String name();
 
   @GraphQLField
   @GraphQLName(SPAN_FILTER_KEY)
+  @GraphQLDescription("Span processing rule filter contains filters based on span attributes")
   @GraphQLNonNull
   SpanProcessingRuleFilter spanFilter();
 
   @GraphQLField
   @GraphQLName(DISABLED_KEY)
+  @GraphQLDescription("Disabled field denotes whether the rule is disabled or not.")
   @GraphQLNonNull
   boolean disabled();
 
   @GraphQLField
   @GraphQLName(RULE_TYPE_KEY)
+  @GraphQLDescription(
+      "Exclude span rule type tells us whether it is user configured rule or system rule")
   // TODO: make this field non-nullable
```

Comment:
Is this a change we still want to make? Anything stopping us from doing it? It can't be done once the API is exposed as it'll be a breaking change.